### PR TITLE
Create "bonk" command

### PR DESCRIPTION
**Pull request by M4TY**

### CODE REVIEW

**Comment by M4TY** [comment]
https://github.com/ssps-cajthaml/discord-bot/issues/14

File: src/command/commands/bonk.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        };
+
+        await message.react('👍');
+
+        const collector = message.createReactionCollector({ filter, time: 60000 });
+
+        collector.on('collect', (reaction: any, user: any) => {
+            console.log(`Collected ${reaction.emoji.name} from ${user.tag}`);
+            if(reaction.count >= requiredVotes) {
+                // @ts-ignore
```
Comment (cajthamlmatej) [inline src/command/commands/bonk.ts:62]: Why?

File: src/command/commands/bonk.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+        const filter = (reaction: any, user: any) => {
+            return reaction.emoji.name === '👍' && !user.bot;
+        };
+
+        await message.react('👍');
+
+        const collector = message.createReactionCollector({ filter, time: 60000 });
+
+        collector.on('collect', (reaction: any, user: any) => {
```
Comment (cajthamlmatej) [inline src/command/commands/bonk.ts:59]: any? we have these typed :(

**Review by cajthamlmatej — CHANGES_REQUESTED** [review]

File: src/command/commands/bonk.ts (hunk 1 above)
Comment (M4TY) [reply src/command/commands/bonk.ts]: Resolved with latest commit.

File: src/command/commands/bonk.ts (hunk 2 above)
Comment (M4TY) [reply src/command/commands/bonk.ts]: Resolved with latest commit.

**Review by cajthamlmatej — APPROVED** [review]

**Pull request merged**